.husky/pre-commit

```diff
@@ -1,4 +1 @@
-#!/bin/sh
-. "$(dirname "$0")/_/husky.sh"
-
-# npx lint-staged
+npm run lint:changed-feature-branch
```

Comment:
are we still going to use both eslint and prettier? Can we remove prettier from package.json if not using?

Reply:
I could add `npm run format` here too. Forgot that we had that honestly.

Reply:
Okay, so that doesn't work as a pre-commit hook. Since things change it kicks you back to the point where everything is just staged and you can't actually commit anything. Since we haven't been using it, I'm just going to remove it for now. We'll have prettier in the git history if we ever want it back.

Reply:
Got it. Approving this.